Resources/Data Sets/index_nl.json

```
@@ -779,6 +779,20 @@
           "description": "Indicatoren van industrialisatie in 1960, en politieke democratie in 1960 en 1965. <br><br> Het voorbeeldbestand in JASP laat zien hoe structural equation modeling kan worden toegepast. <br><br> <i>Dataset in R.<\/i>",
           "kind": "file",
           "associated_datafile": "Political Democracy.csv"
+        },
+        {
+          "name": "Grand Point Average",
+          "path": "Grand Point Average.jasp",
+          "description": "Grand Point Average of 200 college students across 6 semesters, along with information on gender, high school GPA and whether they were admitted to their university of choice. <br><br> The JASP file currently does not contain any analysis. <br><br> <i>Data from Hox, Moerbeek and Van de Schoot (2017).<\/i>",
```

```suggestion
          "description": "Gemiddeld aantal studiepunten van 200 studenten over 6 semesters, samen met informatie over geslacht, aantal studiepunten op de middelbare school en of ze zijn toegelaten tot de universiteit van hun keuze. <br><br>Het JASP-bestand bevat momenteel geen analyses. <br><br> <i>Dataset van Hox, Moerbeek, en Van de Schoot (2017).<\/i>",
```
I also added an Oxford comma.